test/externalTests/gp2.sh

```diff
@@ -69,6 +69,8 @@ function gp2_test
     force_hardhat_unlimited_contract_size "$config_file" "$config_var"
     npm install
 
+    npm install hardhat@2.10.2
```

Comment:
This should really have some comment on why we're doing this.

Reply:
Lol, knew I forgot something, even described the issue in the opening PR comment. Will update tomorrow.